themes/mongodb/src/js/componentFeedback.js

```diff
@@ -1,44 +1,36 @@
-import {Deluge} from 'rigning';
+import deluge from '../deluge/deluge';
 
 let project = null;
+let ratingPanelElement = null;
 
 // Files on which we should not have feedback widgets
 const blacklist = {
     'meta/404': true,
     'search': true
 };
 
-function loadPage() {
+function getPageName() {
     const bodyElements = document.getElementsByClassName('body');
-    if (!bodyElements.length) { return; }
+    if (!bodyElements.length) { return null; }
 
     const pagename = bodyElements[0].getAttribute('data-pagename');
     if (Object.prototype.hasOwnProperty.call(blacklist, pagename)) {
```

Comment:
`includes()` would require a polyfill, and this would involve two passes over the input list and a new array.

Changing `blacklist` to an array and using `blacklist.indexOf(pagename) >= 1`